robozonky-strategy-natural/src/main/java/com/github/robozonky/strategy/natural/conditions/AbstractStoryCondition.java

```diff
@@ -36,7 +36,7 @@ protected AbstractStoryCondition(final Predicate<String> storyLength) {
     @Override
     public boolean test(final Wrapper<?> loan) {
         final String story = loan.getStory();
-        final boolean isStoryProvided = (story == null);
-        return storyLength.test(isStoryProvided ? "" : loan.getStory().trim());
+        final boolean isStoryNotProvided = (story == null);
+        return storyLength.test(isStoryNotProvided ? "" : loan.getStory().trim());
```

Comment:
Good catch. 
Just to satisfy my curiosity: what were you looking for so deep in the code?

Reply:
Also, if we're touching this, would you mind inverting the condition to `isStoryProvided`?
Will read better that way.

Reply:
Inverted.
FYI: I would like to add condition "story contains string ..."

Reply:
OK, let me know if you need any help.